coc/raid.py

```diff
@@ -427,14 +428,15 @@ class RaidLogEntry:
                  "_response_retry"
                  )
 
-    def __init__(self, *, data, client, **kwargs):
+    def __init__(self, *, data, client, clan_tag, **kwargs):
```

Comment:
I believe there is some place in entry_logs where you need to pass this parameter in. Not sure, but your tests will tell if you include normal raid log iteration

Reply:
why do wee need this clan_tag attribute at all?

Reply:
I think because `EventsClient._raids` is a dict that uses the tag as key. See line 723 and the following in events.py for example.